gamemode/modules/police/sv_init.lua

```diff
@@ -370,7 +370,7 @@ function DarkRP.hooks:playerUnArrested(ply, actor)
     end
 
     gamemode.Call("PlayerLoadout", ply)
-    if GAMEMODE.Config.telefromjail then
+    if teleportOverride or (GAMEMODE.Config.telefromjail and teleportOverride ~= false) then
```

Comment:
With a boolean `teleportOverride` it is not possible to set a custom respawn position through this function. Though arguably this could be implemented as a `ply:SetPos` _after_ calling the `ply:unArrest()`.

Reply:
I think this could be used to set a custom respawn after being unjailed if you simply force unarrest respawn off and then set the pos right after. This would allow people to do things like bail someone out and teleport the player to them after being bailed out for example.